Week2/homework/maartjes_work.js

```diff
@@ -1,24 +1,22 @@
-'use strict';
-
-const monday = [
```

Comment:
It seems you have deleted ```const moday = [``` and replaced it by ```const tasks = monday.concat(tuesday);``` by mistake, i'm sure, which generates a syntax error in the file. If you add this back it works!